index.html

```diff
@@ -453,17 +427,12 @@ <h2>6. Send the Message:</h2>
                             output('salt',
                                 base64url.encode(crypto.getRandomValues(
                                                         new Uint8Array(16))));
-                            try {
-                                // Auth key allows for improved Crypto-Key support.
-                                // However, it's being slowly rolled out.
-                                let authKey = subscription_info.getKey('auth');
-                                output('authKey', base64url.encode(authKey));
-                                if (! authKey) {
-                                    console.warn ("This browser doesn't support auth key. Please update.");
-                                }
-                            } catch (e) {
-                                //throw err;
-                                console.warn("Crypto-Key support unavailable.");
+                            // Auth key allows for improved Crypto-Key support.
+                            // However, it's being slowly rolled out.
+                            let authKey = subscription_info.getKey('auth');
```

Comment:
Heh, I really ought to just drop this.
Chrome will never include the same Crypto-Key support, and it's one of the main reasons that this page only works with Gecko browsers.